notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Nit: BigQuery DataFrames, not BigFrames</p><p> </p><p>Nit: Use 2nd person, "Let's" is not recommended per the style guide.</p><p> </p><p>I'm not sure we need to include the "CRUD" sentence. That might be a good reminder / way to organize the samples but I don't think it'll mean much to the average data scientist.</p><p> </p><p>I suggest:</p><p> </p><h2>Use native BigQuery functions on array data</h2><p> </p><p>While Pandas offers vectorized operations and lambda expressions to manipulate array data, BigQuery DataFrames leverages BigQuery's computational power.</p><p> </p><p>Use the <code><a href="https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.bigquery" target="_blank">bigframes.bigquery</a></code>  (abbreviated <code>bbq</code> in the following code samples) package to access to a variety of native BigQuery array operations, such as <a href="https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.bigquery#bigframes_bigquery_array_agg" target="_blank">array_agg</a>, <a href="https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.bigquery#bigframes_bigquery_array_length" target="_blank">array_length</a>, and others.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Good to know. Thanks!</p><p> </p>